pkg/replicate/scheme.go

```diff
@@ -192,7 +198,11 @@ func (rs *replicationScheme) execute(ctx context.Context) error {
 	}
 
 	for id, meta := range metas {
-		if rs.blockFilter(meta) {
+		_, err := rs.fromBkt.ReaderWithExpectedErrs(rs.fromBkt.IsObjNotFoundErr).Get(ctx, path.Join(meta.ULID.String(), metadata.DeletionMarkFilename))
+		if err != nil && !rs.fromBkt.IsObjNotFoundErr(err) {
+			level.Debug(rs.logger).Log("msg", "failed to read deletion mark from origin bucket", "error", err)
```

Comment:
Is this an error, error? Or is this more like 'it could be there, which is an acceptable use-case'? - I'm questioning if this then needs to be Debug log level, info or perhaps even being a fatal error then?